conjur/login/login_logic.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+
+"""
+LoginLogic module
+
+This module is the business logic for writing user-specific configuration
+information to the user's machine
+"""
+
+# Builtins
+import logging
+
+# Internals
+import conjur.constants
+from conjur.endpoints import ConjurEndpoint
+from conjur.http import invoke_endpoint, HttpVerb
+
+class LoginLogic:
+    """
+    LoginLogic
+
+    This class holds the business logic for populating the
+    netrc configuration details needed to login to Conjur
+    """
+    credentials = None
+
+    def __init__(self, credentials):
+        self.credentials = credentials
+
+    @classmethod
+    def get_api_key(cls, ssl_verify, credential_data, password, loaded_conjurrc):
```

Comment:
I think `conjurrc` is a dependency of this command.
Treat it as such, by initializing it with the dependencies, and passing it to both controller and logic in their ctors.